agent.go

```diff
@@ -97,11 +99,14 @@ func newAgent(serviceName, host string, port int, logger LeveledLogger) *agentS
 
 	logger.Debug("initializing agent")
 
+	announceTimeout := 15 * time.Second
```

Comment:
I guess this can be inlined now, right? Shall we introduce constants for both timeouts?

Reply:
👍